settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
I was confused about this as well. If you get it on the path or not on the path. Can you update the readme and add another readme to this module to explain this confusion or to explain why it's on the path but not actually depended on